roles/os10_system/templates/os10_system.j2

```diff
@@ -37,6 +37,13 @@ os10_system:
 {% if os10_system.hostname is defined and os10_system.hostname %}
 hostname {{ os10_system.hostname }}
 {% endif %}
+{% if os10_system.hardware_forwarding_ipv6_extended_prefix is defined %}
+  {% if os10_system.hardware_forwarding_ipv6_extended_prefix is defined %}
```

Comment:
Configuration of extended-prefix is working fine.
Can you please mention how to specify in yaml for no-form of this command?
With this change, looks like, "no hardware l3 ipv6-extended-prefix" is not hit when I specified null value in yaml like below,

hardware_forwarding: scaled-l3-hosts
hardware_forwarding_ipv6_extended_prefix:

Kindly verify the no-form of the command.

Reply:
Sorry for the slow response to this. I think I had it working so that removing the option from the yaml just ran the 'no' form of the command, but I need to double check. I'll let you know and maybe adjust the Jinja template if necessary!

Reply:
@Appal5 : I have now fixed this and made it behave as you would expect. If you test again, you'll find that setting `hardware_forwarding_ipv6_extended_prefix:` will remove the setting on the switch.

Reply:
Thank you for fixing the issue. 'no' form of the command is working fine now.